src/screens/BB-BUS/DetailScreen.js

```diff
@@ -136,13 +110,15 @@ export const DetailScreen = ({ route }) => {
 
   const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp, refreshTime });
 
-  const { data, loading, refetch } = useQuery(GET_DIRECTUS, {
-    variables: GET_SERVICE(id),
+  const areaId = secrets[namespace]?.busBb?.areaId;
```

Comment:
ah i think this is the reason for https://github.com/ikuseiGmbH/smart-village-app-app/pull/419/files#r784222846.
here the area id from secrets is used instead of a other possibly selected one, right?

Reply:
this should be fixed now.